service/src/main/java/com/esm/epam/util/CurrentDate.java

```diff
@@ -5,10 +5,11 @@
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 
-import static com.esm.epam.util.ParameterAttribute.DATE_PATTERN;
 
 @Component
 public class CurrentDate {
```

Comment:
If you define in configuration Bean objectMapper and setup it you will able to delete this class and also, I think annotation which you use @Json.View also become redundant